app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityDebugTest.kt

```diff
@@ -315,15 +316,24 @@ class NavigationDrawerActivityDebugTest {
       onView(withId(R.id.developer_options_linear_layout)).perform(nestedScrollTo())
         .check(matches(isDisplayed())).perform(click())
       intended(hasComponent(DeveloperOptionsActivity::class.java.name))
-      intended(
-        hasExtra(
-          DeveloperOptionsActivity.getIntentKey(),
-          internalProfileId
-        )
-      )
+      it.onActivity { it1 ->
+        Truth.assertThat(
```

Comment:
Nit: please directly import assertThat. Ditto below & elsewhere in the PR.

Reply:
Updated.